remote_deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+wget https://github.com/ONSdigital/eq-submitter/archive/master.zip
+unzip master.zip
+mv eq-submitter-master eq-submitter
+cd eq-submitter
+sudo pip install -r requirements.txt
+sudo service supervisor restart
```

Comment:
This doesn't copy the file as explained in the README - does this matter? Also this doesn't install the necessary packages by apt-get - can we add that in here so we can automate all of it?